bash/dev_compile_python_packages.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Dev script to compile python packages from a requirements.in file to a requirements.txt file.
+set -e
+RELATIVE_SCRIPTPATH=$0
+
+# Update and install packages used to compile requirements
+python3 -m pip install --upgrade pip
+python3 -m pip install --upgrade pip-tools wheel
+
+# Delete existing requirements file to ensure full dependency resolution
+rm -rf requirements.txt
+
+# Compile requirements
+CUSTOM_COMPILE_COMMAND="${RELATIVE_SCRIPTPATH}" python3 -m piptools compile --output-file=requirements.txt requirements.in
```

Comment:
I know you've used this before, but didnt' ask then - what's the `RELATIVE_SCRIPTPATH` part of this for? Rather than just starting the final line with the `python3` command